services/brig/test/resources/turn/servers-v2.txt

```diff
@@ -1,2 +1 @@
-turn:localhost:3478
-turn:localhost:3479
+turn:localhost:3478
```

Comment:
what is this about?  can't find anything in the commit messages.

Reply:
Ugh; this happens when you cancel test runs before they complete; it sneaks in for me ALL the time. Thanks